tests/integration/observer_compatibility_empty/resources/taste/work/modelchecking/properties/observer/observer.pr

```diff
@@ -24,23 +26,39 @@ system observer;
             errorstates error;
             successstates reached;
             /* CIF ENDTEXT */
+            /* CIF procedure (72, 421), (70, 35) */
+            /* CIF Keep Specific Geode Partition 'default' */
```

Comment:
Why this change? I though we may even drop the translation of Observable_Event....

Reply:
Well, we had a discussion about it and the conclusion was that the the procedure is not required to translate observer with sdl2promela.
However, after some time I realised that I added `observe` procedures for tests with `_compatibility_` in name.
Those tests check if SDL created using space creator can be translated using sdl2promela.
At this moment I have no idea why those procedures are missing, there is only a declaration, but no body.

I suggest that empty procedure `observe` is required here.